src/license-comment.h

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2016-2020 The CRoaring authors
```

Comment:
This is a guess from what I found in the source code - I first saw this project a few days ago while trying to package a project that includes an amalgamated copy, and do not have in-depth knowledge of it. If there's something better to put here, please suggest something or edit the branch.

Reply:
This is good.